cmd/ipfs/goreq.go

```diff
@@ -0,0 +1,3 @@
+// +build !go1.5		
+
+`IPFS needs to be built with go version 1.5 or greater`
```

Comment:
True, I just feared tools like `go list` stumbling over this. Also: what happens once go1.6 is out?